### ISSUE TITLE
Add error handling to all queries.

### ISSUE THREAD

**Issue by rutgerfarry** [classmere/api#7]
If most queries in the app fail, the app will crash. We just need to log these errors and continue.
To do this, just add the following in each routing function right after a query is made:

``` javascript
query.on('error', function(err) {
  console.error(err);
});
```

**Comment by rutgerfarry** [comment]
we should actually send an error to the user too. not `err` but just something like `res.json(500, 'internal error'`

**Comment by rutgerfarry** [comment]
Closed by 9c8b65e
